ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -840,6 +846,12 @@ bool UncoupledProcessesTimeLoop::solveUncoupledEquationSystems(
             _output->doOutputAlways(pcs, spd->process_output, timestep_id, t,
                                     x);
 
+            auto& timestepper = spd->timestepper;
+            if (!timestepper->isSolutionErrorComputationNeeded())
```

Comment:
There are only two time stepping algorithms available. If more  time stepping algorithms added, this condition will be changed.